slurp/src/main/webapp/scripts/utils.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Parse URL <pre><code>NAME.html?key1=value1&key2=value2...</code></pre> into {key1: value1, ...}.
+ * @param {string} url The URL to be parsed. 
+ * @returns {Object.<string, string>} Dictionary of the parsed URL parameters. 
+ */
+function parseUrl(url) {
+  var paramsUrl = url.substring(url.indexOf("?") + 1);
+  var paramPairs = paramsUrl.split("&");
+  
+  var params = {};
+  for (var pair of paramPairs) {
+    var splitIndex = pair.indexOf("=");
```

Comment:
Change all string literals to use single quote rather than double based on [this](https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes). I didn't want to clutter up the comments with multiple of these comments so make sure to check your other js files for this.